integration_tests/nrp/nr.py

```diff
@@ -0,0 +1,20 @@
+from ltypes import f64, i32
+
+
+def func(x: f64, c: f64) -> f64:
+	return x**2.0 - c**2.0
+
+
+def func_prime(x: f64, c: f64) -> f64:
+	return 2.0*x
+
+
+def newton_raphson(x: f64, c: f64, maxiter: i32) -> f64:
+	h: f64 = func(x, c) / func_prime(x, c)
+	err: f64 = 1e-5
+	i: i32 = 0
+	while abs(func(x, c)) > err and i < maxiter:
+		h = func(x, c) / func_prime(x, c)
+		x = x - h
+		i += 1
```

Comment:
Convert tabs to spaces.

Reply:
Ah, some wierd issue. I have used a tab for indenting and locally looks fine.

Reply:
We should probably set some pre-commit hooks to autoformat the code when commiting. we can use python black as the formatter.

Reply:
We should use LPython as the formatter. :)

Reply:
Yup!